### DISCUSSION TITLE
Transaction not found on etherscan.io after a successful transaction

### DISCUSSION THREAD

**Discussion by maheshbsl** [Cyfrin/foundry-full-course-cu#2520] [General]
After deploying a smart contract successfully on sepolia testnet , when I try searching that transaction hash on ethersan , there is not a transaction.There are requests on alchemy though.

This is the success message
```
##### sepolia
✅  [Success]Hash: 0xe297c14dd6daadb6823ab3c725f71385210287fc619494c0d194317fd621c5f3
Contract Address: 0x5376B0a1ea7DD19330266Fe430756c9542e6de91
Block: 6731446
Paid: 0.005629534399766172 ETH (357252 gas * 15.757880711 gwei)

✅ Sequence #1 on sepolia | Total Paid: 0.005629534399766172 ETH (357252 gas * avg 15.757880711 gwei)
    ```

**Comment by maheshbsl** [comment]
After 30 minutes, I figure out that I was searching the transaction on the Ethereum Main Net instead of Sepolia Test Net.

**Reply by EngrPips** [reply]
Interesting, we are glad you figured that out sir, Keep crushing it.
